### PR TITLE
[multitenancy-manager] Default project template

### PR DESCRIPTION
**Pull request by Dmitrykob**
## Description

Added default project template for all users as example

## Why do we need it, and what problem does it solve?

  Fixes an issue https://github.com/deckhouse/deckhouse/issues/5614

## What is the expected result?

New default project template for all users

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: multitenancy-manager
type: fix
summary: Added default project templates: **default** to match most cases and **secure** for advanced secured projects.
impact_level: default
```

### CODE REVIEW

File: ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      name: all-containers
+    spec:
+      limits:
+        - max:
+            {{ with .templateValues.limits.cpu }}cpu: {{ . }}{{ end }}
+            {{ with .templateValues.limits.memory }}memory: {{ . }}{{ end }}
+          maxLimitRequestRatio:
+            cpu: 1
+            memory: 1
+          type: Container
```
Comment (nabokihms) [inline ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml:84]: Let's not create the limit range for now. The behavior of this resource is a real puzzle.
```suggestion
```

Instead, let's provide an example with the operation policy to enforce setting requests by a user.
https://deckhouse.io/documentation/v1/modules/015-admission-policy-engine/cr.html#operationpolicy-v1alpha1-spec-policies-requiredresources

File: ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      podSelector:
+        matchLabels: {}
+      policyTypes:
+        - Ingress
+        - Egress
+      ingress:
+        - from:
+            - namespaceSelector:
+                matchLabels:
+                  kubernetes.io/metadata.name: "{{ .projectName }}"
```
Comment (nabokihms) [inline ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml:102]: With the current ingress policy, Prometheus will not be able to scrape metrics. Let's add the rule for it:
```yaml
- namespaceSelector:
    matchLabels:
       kubernetes.io/metadata.name: "d8-monitoring"
   podSelector:
     matchLabels:
       app: prometheus
```

**Review by nabokihms — CHANGES_REQUESTED** [review]
This is a good start. Yet let's add more features to the default template. Users need to know which features they can use.

I propose adding **three** templates:

1. `default` template with the following options:

Quota
```
resourceQuota:
  requests: (optional)
    cpu (optional)
    memory (optional)
    storage (optional)
  limits (optional)
    cpu (optional)
    memory (optional)
```

create network policy or not
```
networkPolicy: Isolated (default) | NotRestricted
```

Pod security policy level (the value for the `security.deckhouse.io/pod-policy=<POLICY_NAME>` label)
```
podSecurityProfile: Privileged | Baseline (default) | Restricted
```
Extended monitoring (set the label `extended-monitoring.deckhouse.io/enabled`  on the namespace or not)
```
extendedMoniroting: true (default) | false
```
The value for the clusterlogdestination in the podloggingconfig ([there](https://deckhouse.io/documentation/v1/modules/460-log-shipper/examples.html#creating-a-source-in-namespace-and-reading-logs-of-all-pods-in-that-ns-with-forwarding-them-to-loki) you can find an example)
```
clusterLogDestnationName: string (optional)
```

Who has the access to the project
```
administrators:
- subject: User | Group
  name: "..."
```

2. `secure` - the same as default, but with additional options.

Trivy scanner (set the label `security-scanning.deckhouse.io/enabled` on the namespace or not) 
```
securityScanning: true (true) | false
```
Set allowed uids/guids with [the SecurityPolicy](https://deckhouse.io/documentation/v1/modules/015-admission-policy-engine/cr.html#securitypolicy)
```
allowedUIDs: (optional)
  max:  (required)
  min: (required)
allowedGIDs: (optional)
  max:  (required)
  min: (required)
```  
Enable audit rules (create a falco audit rule to record some malicious activities of the pods in a given namespace)
```
enabledAudit: true | false (default)
```

3. `secure-with-dedicated-nodes` - the same as secure, but with options for dedicated nodes.

```
dedicatedNodes:
  labelSelector: {...}
  tolerations: [{...}]
```

**Review by MariaMakeeva — APPROVED** [review]

**Review by MariaMakeeva — APPROVED** [review]

File: ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml [hunk 3]
```diff
@@ -0,0 +1,170 @@
+---
+apiVersion: deckhouse.io/v1alpha1
+kind: ProjectTemplate
+metadata:
+  name: default-project-template
```
Comment (nabokihms) [inline ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml:5]:
```suggestion
  name: default
```

File: ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    ---
+    # Loki ClusterLogDestination config example
+    # apiVersion: deckhouse.io/v1alpha1
+    # kind: ClusterLogDestination
+    # metadata:
+    #   name: loki-storage
+    # spec:
+    #   type: Loki
+    #   loki:
+    #     endpoint: http://loki.loki:3100
```
Comment (nabokihms) [inline ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml:170]:
```suggestion
```

File: ee/modules/160-multitenancy-manager/templates/user-resources/secure-project-template.yaml [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+      - rule:
+          name: Detect Ownership Change
+          desc: detect file permission/ownership change
+          condition: >
+            spawned_process and proc.name in (chmod, chown) and proc.args contains "/tmp/"
+          output: >
+            The file or directory below has had its permissions or ownership changed (user=%user.name
+            command=%proc.cmdline file=%fd.name parent=%proc.pname pcmdline=%proc.pcmdline gparent=%proc.aname[2])
+          priority: Warning
+          tags: [filesystem]
```
Comment (nabokihms) [inline ee/modules/160-multitenancy-manager/templates/user-resources/secure-project-template.yaml:225]: We need to add something more meaningful. Let's think about it.

File: ee/modules/160-multitenancy-manager/templates/user-resources/secure-project-template.yaml [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+          Extended monitoring (set the label extended-monitoring.deckhouse.io/enabled on the namespace or not).
+        type: boolean
+        default: true
+      clusterLogDestnationName:
+        description: |
+          The value for the clusterlogdestination in the podloggingconfig.
+        type: string
+      securityScanning:
+        description:
+          Trivy scanner.
```
[the remark this answers is not in the document]
Comment (Dmitrykob) [reply ee/modules/160-multitenancy-manager/templates/user-resources/secure-project-template.yaml]: Added a more detailed description

File: ee/modules/160-multitenancy-manager/templates/user-resources/secure-project-template.yaml (hunk 5 above)
Comment (Dmitrykob) [reply ee/modules/160-multitenancy-manager/templates/user-resources/secure-project-template.yaml]: Replaced the template with an example of sending messages about shell is run in a container

File: ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml (hunk 2 above)
Comment (Dmitrykob) [reply ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml:135]: implemented

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by MariaMakeeva — APPROVED** [review]

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 7]
```diff
@@ -1,290 +1,80 @@
...
-   Проверьте, что окружение успешно создалось, выполнив следующую команду:
-
-   ```shell
-   kubectl get projects.deckhouse.io
-   ```
-
-   Пример вывода:
+## Шаблоны для проектов доступные по умолчанию
+
+В поставку Deckhouse Kubernetes Platform включены следующие шаблоны для проектов:
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:8]:
```suggestion
Deckhouse Kubernetes Platform включает следующий набор шаблонов для создания проектов:
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 8]
```diff
@@ -1,290 +1,80 @@
...
+- `default` — шаблон, который покрывает базовые сценарии использования проектов:
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:9]:
```suggestion
- `default` — шаблон для базовых сценариев использования проектов:
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 9]
```diff
@@ -1,290 +1,80 @@
...
+  * ограничение ресурсов;
+  * сетевая изоляция;
+  * автоматические алерты и сбор логов;
+  * выбор профиля безопасности;
+  * настройка администраторов проекта.
+
+- `secure` — содержит в себе все возможности шаблона `default` и дополнительно:
+  * настройка допустимых для проекта UID/GID;
+  * правила аудита обращения к ядру линукс пользоваетелей проекта;
+  * сканирование запускаемых образов контейнеров на наличие CVE.
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:19]:
```suggestion
  * настройку допустимых для проекта UID/GID;
  * правила аудита обращения к ядру линукс пользоваетелей проекта;
  * сканирование запускаемых образов контейнеров на наличие CVE.
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 10]
```diff
@@ -1,290 +1,80 @@
...
+
+## Создание проекта
+
+Для создания проекта необходимо создать ресурс [Project](cr.html#project) с указанием имени шаблона проекта в поле [.spec.projectTemplateName](cr.html#project-v1alpha1-spec-projecttemplate).
+В поле [.spec.template](cr.html#project-v1alpha1-spec-template) ресурса `Project` необходимо указать значения параметров, которые подходят для [.spec.schema.openAPIV3Schema ProjectTemplate](cr.html#projecttemplate-v1alpha1-spec--schema-openAPIV3Schema).
+
+Пример создания проекта с помощью ресурса [Project](cr.html#project) из `default` [ProjectTemplate](cr.html#projecttemplate):
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:26]:
```suggestion
Пример создания проекта с помощью ресурса [Project](cr.html#project) из `default` [ProjectTemplate](cr.html#projecttemplate) представлен ниже:
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 11]
```diff
@@ -1,290 +1,80 @@
...
+  * настройка администраторов проекта.
+
+- `secure` — содержит в себе все возможности шаблона `default` и дополнительно:
+  * настройка допустимых для проекта UID/GID;
+  * правила аудита обращения к ядру линукс пользоваетелей проекта;
+  * сканирование запускаемых образов контейнеров на наличие CVE.
+
+## Создание проекта
+
+Для создания проекта необходимо создать ресурс [Project](cr.html#project) с указанием имени шаблона проекта в поле [.spec.projectTemplateName](cr.html#project-v1alpha1-spec-projecttemplate).
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:23]:
```suggestion
1. Для создания проекта создайте ресурс [Project](cr.html#project) с указанием имени шаблона проекта в поле [.spec.projectTemplateName](cr.html#project-v1alpha1-spec-projecttemplate).
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 12]
```diff
@@ -1,290 +1,80 @@
...
+В поле [.spec.template](cr.html#project-v1alpha1-spec-template) ресурса `Project` необходимо указать значения параметров, которые подходят для [.spec.schema.openAPIV3Schema ProjectTemplate](cr.html#projecttemplate-v1alpha1-spec--schema-openAPIV3Schema).
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:24]:
```suggestion
2. В поле [.spec.template](cr.html#project-v1alpha1-spec-template) ресурса `Project` укажите значения параметров для [.spec.schema.openAPIV3Schema ProjectTemplate](cr.html#projecttemplate-v1alpha1-spec--schema-openAPIV3Schema).
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 13]
```diff
@@ -1,290 +1,80 @@
...
+        storage: 1Gi
+      limits:
+        cpu: 5
+        memory: 5Gi
+    networkPolicy: Isolated
+    podSecurityPolicy: Restricted
+    enableExtendedMonitoring: true
+```
+
+Чтобы посмотреть статус проекта, выполните команду:
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:50]:
```suggestion
3. Для проверки статуса проекта введите команду:
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 14]
```diff
@@ -1,290 +1,80 @@
...
+
+```shell
+kubectl get projects my-project
+```
+
+Успешно созданный проект должен быть в статусе "Синзронизирован".
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:56]:
```suggestion
Успешно созданный проект должен отображаться в статусе "Синзронизирован".
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 15]
```diff
@@ -1,290 +1,80 @@
...
+
+## Создание своего шаблона для проекта
+
+Шаблоны по умолчанию содержат в себе базовые сценарии использования проектов и служат хорошим примером применения возможностей шаблонизации.
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:60]:
```suggestion
Шаблоны проектов по умолчанию включают базовые сценарии использования и служат примером возможностей шаблонов.
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 16]
```diff
@@ -1,290 +1,80 @@
...
-   ```shell
-   $ kubectl get -n test-project namespaces test-project
-   NAME           STATUS   AGE
-   test-project   Active   5m
-
-   $ kubectl get authorizationrules.deckhouse.io -n test-project
-   NAME                                    AGE
-   test-project-admin-user-admin-cluster   5m
-   test-project-user-user-user-cluster     5m
+3. Отредактируйте файл `my-project-template.yaml`, внесите в него необходимые изменения. **Не забудьте** что изменить нужно не только шаблон, но и схему входных параметров под него.
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:70]:
```suggestion
3. Отредактируйте файл `my-project-template.yaml`, внесите в него необходимые изменения. 
> Примечание: необходимо изменить не только шаблон, но и схему входных параметров под него.
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 17]
```diff
@@ -1,290 +1,80 @@
...
+4. Измените имя шаблона в поле [.metadata.name](cr.html#projecttemplate-v1alpha1-metadata-name).
+5. Примените свой новый шаблон командой:
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:72]:
```suggestion
5. Примените полученный шаблон командой:
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 18]
```diff
@@ -1,290 +1,80 @@
...
-   ```shell
-   $ kubectl get limitranges -n test-project --kubeconfig admin-kubeconfig.yaml
-   NAME                          CREATED AT
-   test-project-all-containers   2023-06-01T14:37:42Z
-
-   $ kubectl get limitranges -n test-project --kubeconfig user-kubeconfig.yaml
-   NAME                          CREATED AT
-   test-project-all-containers   2023-06-01T14:37:42Z
-   ```
+> Шаблоны для проектов поддерживают все [функции шаблонизации Helm](https://helm.sh/docs/chart_template_guide/function_list/).
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:78]:
```suggestion
> Примечание: шаблоны для проектов поддерживают все [функции шаблонизации Helm](https://helm.sh/docs/chart_template_guide/function_list/).
```

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 19]
```diff
@@ -1,290 +1,80 @@
...
+
+Чтобы посмотреть статус проекта, выполните команду:
+
+```shell
+kubectl get projects my-project
+```
+
+Успешно созданный проект должен быть в статусе "Синзронизирован".
+
+## Создание своего шаблона для проекта
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:58]: Может быть "своего" заменить на "пользовательского" или "собственного"?

File: ee/modules/160-multitenancy-manager/docs/USAGE.md [hunk 20]
```diff
@@ -1,290 +1,80 @@
...
+
+The following project templates are included in the Deckhouse Kubernetes Platform:
+- `default` — a template that covers basic project use cases:
+  * resource limitation;
+  * network isolation;
+  * automatic alerts and log collection;
+  * choice of security profile;
+  * project administrators setup.
+
+- `secure` — includes all the capabilities of the `default` template and additionally:
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE.md:16]:
```suggestion
- `secure` — includes all the capabilities of the `default` template and additional features:
```

File: ee/modules/160-multitenancy-manager/docs/USAGE.md [hunk 21]
```diff
@@ -1,290 +1,80 @@
...
+  * setting up permissible UID/GID for the project;
+  * audit rules for project users' access to the Linux kernel;
+  * scanning of launched container images for CVE presence.
+
+## Creating a project
+
+To create a project, you need to create a [Project](cr.html#project) resource specifying the name of the project template in the [.spec.projectTemplateName](cr.html#project-v1alpha1-spec-projecttemplate) field.
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE.md:23]:
```suggestion
To create a project, create the [Project](cr.html#project) resource by specifying the name of the project template in  [.spec.projectTemplateName](cr.html#project-v1alpha1-spec-projecttemplate) field.
```

File: ee/modules/160-multitenancy-manager/docs/USAGE.md [hunk 22]
```diff
@@ -1,290 +1,80 @@
...
+In the [.spec.template](cr.html#project-v1alpha1-spec-template) field of the `Project` resource, you need to specify the parameter values that are suitable for the [.spec.schema.openAPIV3Schema ProjectTemplate](cr.html#projecttemplate-v1alpha1-spec--schema-openAPIV3Schema).
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE.md:24]:
```suggestion
2. In the [.spec.template](cr.html#project-v1alpha1-spec-template) field of the `Project` resource, specify the parameter values suitable for the [.spec.schema.openAPIV3Schema ProjectTemplate](cr.html#projecttemplate-v1alpha1-spec--schema-openAPIV3Schema).
```

File: ee/modules/160-multitenancy-manager/docs/USAGE.md [hunk 23]
```diff
@@ -1,290 +1,80 @@
...
-   ```shell
-   $ kubectl get -n test-project namespaces test-project
-   NAME           STATUS   AGE
-   test-project   Active   5m
-
-   $ kubectl get authorizationrules.deckhouse.io -n test-project
-   NAME                                    AGE
-   test-project-admin-user-admin-cluster   5m
-   test-project-user-user-user-cluster     5m
+3. Edit the `my-project-template.yaml` file, make the necessary changes. **Don't forget** that you need to change not only the template but also the schema of input parameters for it.
```
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE.md:70]:
```suggestion
3. Edit the `my-project-template.yaml` file, make the necessary changes. 

> Note: it is necessary to change not only the template, but also the scheme of input parameters for it.
```

**Review by MariaMakeeva — COMMENTED** [review]
The English and Russian texts have been corrected.

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 24]
```diff
@@ -1,290 +1,80 @@
...
+
+В поставку Deckhouse Kubernetes Platform включены следующие шаблоны для проектов:
+- `default` — шаблон, который покрывает базовые сценарии использования проектов:
+  * ограничение ресурсов;
+  * сетевая изоляция;
+  * автоматические алерты и сбор логов;
+  * выбор профиля безопасности;
+  * настройка администраторов проекта.
+
+- `secure` — содержит в себе все возможности шаблона `default` и дополнительно:
```
[the remark this answers is not in the document]
Comment (z9r5) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:16]: либо "шаблона `default`", либо "`default`-шаблона".  Я за первый вариант

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 12 above)
Comment (z9r5) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:24]: Давай ссылку поправим (и чуть тексат) 
...укажите значения параметров для секции [.spec.schema.openAPIV3Schema](cr.html#projecttemplate-v1alpha1-spec--schema-openAPIV3Schema) ресурса `ProjectTemplate`.
?

ну и договаривались что все-таки в русской версии будет не поле ресурса а параметр ресурса (в английской - field)

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 25]
```diff
@@ -1,290 +1,80 @@
...
+    networkPolicy: Isolated
+    podSecurityPolicy: Restricted
+    enableExtendedMonitoring: true
+```
+
+Чтобы посмотреть статус проекта, выполните команду:
+
+```shell
+kubectl get projects my-project
+```
```
Comment (z9r5) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:54]: тут  надо отступ не забыть, если список смержим

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 14 above)
Comment (z9r5) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:56]: @Dmitrykob Подскажи, какой реально статус будет ? СинХронизирован или что-то типа  Synchronized?

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (MariaMakeeva) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:16]: Ок)

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 16 above)
Comment (z9r5) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:70]: Строчкус  примечание надо сдвинуть на три пробела

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 18 above)
Comment (z9r5) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:78]: ИМХО нет смысла писать слово "Примечание" - оно и так визуально как примечание сверстано будет.

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 16 above)
Comment (MariaMakeeva) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:70]: Окей, если сверстано будет - то норм. Меня смутил боль - ** не забудьте**

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 18 above)
Comment (MariaMakeeva) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:78]: Ок)

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 12 above)
Comment (MariaMakeeva) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:24]: Да, поправила.

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 14 above)
Comment (Dmitrykob) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:56]:
```
status:
  state: Sync
  sync: true
```

Comment (MariaMakeeva) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:56]: Поправила в связи с данными.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 19 above)
Comment (z9r5) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:58]: IDK. Up to you.

**Review by z9r5 — APPROVED** [review]

File: ee/modules/160-multitenancy-manager/docs/USAGE_RU.md (hunk 19 above)
Comment (MariaMakeeva) [inline ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:58]:
```suggestion
## Создание собственного шаблона для проекта
```

Comment (MariaMakeeva) [reply ee/modules/160-multitenancy-manager/docs/USAGE_RU.md:58]: Готово

**Review by nabokihms — APPROVED** [review]

**Pull request merged**